openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -15,10 +34,53 @@ const WaitingStepList = ({ studentList, refreshData }) => {
     <Button onClick={() => refreshData()}>{gettext('Refresh')}</Button>
   );
 
+  const FindLearnerAction = (props) => {
+    const { tableInstance: { selectedFlatRows = [] } } = props || { tableInstance: {} };
+
+    const selectedFlatRowsLength = selectedFlatRows.length;
+
+    if (selectableLearners && selectedFlatRowsLength) {
```

Comment:
```suggestion
    if (selectableLearners && selectedFlatRows.length) {
```

Reply:
That constant is used [here](https://github.com/openedx/edx-ora2/pull/2025/files#diff-422421bc53f8bd0e9e53aee9916ae3184ef580902fa8fb5b40cad5f5befb0266R48) as well